app/server/src/types.ts

```diff
@@ -9,7 +9,9 @@ export interface WodinConfig {
     baseUrl: string,
     odinApi: string,
     redisUrl: string,
-    appsPath: string
+    appsPath: string,
+    internationalisation: boolean,
```

Comment:
I suggest naming this (and subsequent usees) `i18n` to avoid annoying typing, and the s-vs-z spelling issues. Perhaps your editors are good enough to make that less painful?